store/views.py

```diff
@@ -77,6 +95,36 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    if request.method == "POST":
+        try:
+            bid = request.POST['bid']
+            book = BookCopy.objects.get(pk=bid)
+            book.status = True
+            book.borrower = None
+            book.borrow_date = None
+            book.save()
+            return JsonResponse( {"message":"Book successfully returned."} )
+        except:
+            return JsonResponse( {"message":"No 'bid' found"} )
+    else:
+        return JsonResponse( {"message":"Bad Request"} )
+
+@csrf_exempt
+@login_required
+def rateBookview(request, bid):
+    if request.method == "POST":
+        book = Book.objects.get(pk=bid)
+        rating = request.POST['rating']
```

Comment:
You've not put a backend validation on this value, so the user can simply edit the JS code you've written in the template and easily put invalid values of rating.